chapters/it/_toctree.yml

```diff
@@ -0,0 +1,173 @@
+- title: 0. Installazione
+  sections:
+  - local: chapter0/1
+    title: Introduzione
+
+- title: 1. Modelli Transformer
+  sections:
+  - local: chapter1/1
+    title: Introduzione
+  - local: chapter1/2
```

Comment:
Thanks for translating the whole ToC, but we only need to include the sections for which there are MDX files in Italian. In your case, we have `chapter0/1.mdx` and `chapter1/1.mdx` translated, so the `_toctree.yml` file should just include:

```yaml
- title: 0. Installazione
  sections:
  - local: chapter0/1
    title: Introduzione

- title: 1. Modelli Transformer
  sections:
  - local: chapter1/1
    title: Introduzione
```

Reply:
Ok! Thanks!